app/policies/board_policy.rb

```diff
@@ -34,7 +34,12 @@ def create?
   end
 
   def update?
-    record.next_board_present? ? false : user.allowed?('update_board', record)
+    current_board = Board.includes([:next_board]).find(record.id)
```

Comment:
record is not current board ?

Reply:
current, but I have a bullet error and to fix this I create a new variable current_board, i can rename it

Reply:
but this solution also generate two requests to db